src/plugins/vis_type_table_new/public/types.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Yeah because we are not completely re-writing table vis. We are using some functions and types from previous table. So in this PR, I added `ColumnWidth`, SortColumn and TableUiState. But FormattedColumn, TableVisParams and TableConfig are using old. If there is a combination of old and new, I just use this license.